.github/pull_request_template.md

```diff
@@ -1,30 +1,20 @@
 ## Relevant Issues
-<!-- List relevant issue tickets here. -->
-<!-- Say "Closes #0" for issues that the PR resolves, replacing 0 with the issue number. -->
+<!--
+Linking an issue approved by a Core Developer, which will have the "approved" label, is mandatory.
```

Comment:
I had to read this sentence twice to understand it, does this read better? 
```suggestion
Linking an issue approved by a Core Developer, that has the "approved" label, is mandatory.
```
Maybe the commas aren't necessary too

Reply:
Tbh the wording sounds a bit unnatural in general.

> It is mandatory to link an issue with the "approved" label. 

We don't really need to outline that it's approved by a Core Dev, when the label already says that in it's description, do we?

Reply:
The reason for adding it in the first place was to address concerns about who is approving issues. Currently, anyone with write permissions (all staff members) can add the approved label. The conversation was linked in the issue, but [here](https://discord.com/channels/267624335836053506/411200599653351425/832591634952618004) it is again (staff channel). I don't mind it either way, but the consensus at the time was approving issues is a core dev responsibility, we should outline that.

Reply:
Okay. In that case, it's really only getting rid of the inner phrasing to make it flow better while keeping the same info.

> It is mandatory to link to an issue that has been approved by a Core Developer, indicated by an "approved" label.

Reply:
Addressed in 525078b